ChemistryLib/PhreeqcKernelData/EquilibriumReactants.cpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "EquilibriumReactants.h"
+
+namespace ChemistryLib
+{
+namespace PhreeqcKernelData
+{
+PhaseComponent::PhaseComponent(std::string&& name_,
+                               double const initial_amount,
+                               double const saturation_index)
+{
+    name = name_;
```

Comment:
Minor: A move is need here. Not that it makes any difference in runtime, but, I think, if we use r-value references etc. it should be consistent 'til end.

Reply:
Fixed.